control

```diff
@@ -0,0 +1,20 @@
+Package: ni-veristand-custom-device-support
+Version: {nipkg_version}
+Architecture: windows_all
+Maintainer: National Instruments <support@ni.com>
+XB-Plugin: file
+Description:  Adds custom devices to NI VeriStand.
+  Custom devices are add-ons that provide extended functionality to the NI VeriStand environment.
+XB-Eula:  eula-ni-standard,
```

Comment:
Some of these lines have two leading spaces, and some have one. We should be consistent.